src/app/app.module.ts

```diff
@@ -3,38 +3,53 @@ import { BrowserModule } from '@angular/platform-browser';
 import { IonicApp, IonicModule, IonicErrorHandler } from 'ionic-angular';
 import { MyApp } from './app.component';
 
-import { AboutPage } from '../pages/about/about';
-import { ContactPage } from '../pages/contact/contact';
-import { HomePage } from '../pages/home/home';
-import { TabsPage } from '../pages/tabs/tabs';
+import { WificallPage } from '../pages/wificall/wificall';
```

Comment:
@tokland Can we use upper case for the C in Call? I know I am being a bit picky...